src/ControllerV2/UserController.php

```diff
@@ -96,9 +96,9 @@ public function delete(Request $request, UserManager $userManager, TokenStorageI
 
     #[IsGranted('ROLE_MEMBRE')]
     #[Route(path: '/{id<\d+>}/toggle-invite/{relay<\d+>}', name: 'toggle_user_invitation', methods: ['GET'])]
-    public function toggleUserInvitation(User $user, #[MapEntity(id: 'relay')] Centre $relay, RelayManager $manager): Response
+    public function toggleUserInvitation(Request $request, User $user, #[MapEntity(id: 'relay')] Centre $relay, RelayManager $manager): Response
     {
-        $manager->toggleUserInvitationToRelay($user, $relay);
+        $manager->toggleUserInvitationToRelay($user, $relay, $request->query->getBoolean('forceAffiliation'));
```

Comment:
You sure using a query param to force accept invit is not security failure prone ?